telemetry_core/build.gradle.kts

```
@@ -21,4 +31,18 @@ dependencies {
     testImplementation("com.google.guava:guava:${Versions.guava}")
     testImplementation("org.skyscreamer:jsonassert:${Versions.jsonassert}")
 
-}
\ No newline at end of file
+}
+
+tasks {
+    "shadowJar"(ShadowJar::class) {
+        classifier = ""
+        dependencies {
+            exclude(dependency("org.slf4j:slf4j-api:${Versions.slf4j}"))
+        }
+        relocate("com.google.gson", "a")
```

```suggestion
        relocate("com.google.gson", "com.newrelic.relocated")
```
Just in case something pops up and people wonder where `a` originated from.  At least this could be a hint.